### PR TITLE
PFM-2141 | Expense calculator | update as per businessService names

### PR DESCRIPTION
**Pull request by vishalgupta-eGov**
Please review it

### CODE REVIEW

**Review by subhashini-egov — COMMENTED** [review]
@vishalgupta-eGov I have submitted some things for review. Please see those and close it out.

File: backend/expense-calculator/src/main/java/org/egov/digit/expense/calculator/util/ExpenseCalculatorServiceConstants.java [hunk 1]
```diff
@@ -52,4 +52,5 @@ public class ExpenseCalculatorServiceConstants {
     public static final String LUMPSUM_CONSTANT = "lumpsum";
     public static final String EXPENSE_CONSTANT = "expense";
     public static final String DOCUMENTS_CONSTANT = "documents";
+    public static final String ACTIVE_STATUS_CONSTANT = "ACTIVE";
```
[the remark this answers is not in the document]
Comment (vishalgupta-eGov) [reply backend/expense-calculator/src/main/java/org/egov/digit/expense/calculator/util/ExpenseCalculatorServiceConstants.java:55]: Wage bill initial status while creating from musterroll

File: backend/expense-calculator/src/main/java/org/egov/digit/expense/calculator/util/MdmsUtils.java [hunk 2]
```diff
@@ -33,7 +33,21 @@ public Object fetchMDMSForValidation(RequestInfo requestInfo, String tenantId) {
 
     public Object getPayersForTypeFromMDMS(RequestInfo requestInfo, String type, String tenantId){
         List<MasterDetail> masterDetails = new ArrayList<>();
-        MasterDetail masterDetail = getMasterDetailForSubModule(MDMD_PAYER_LIST, "$.[?(@.active==true && @.type=='"+type+"')]");
+        MasterDetail masterDetail = getMasterDetailForSubModuleAndFilter(MDMD_PAYER_LIST, "$.[?(@.active==true && @.type=='"+type+"')]");
```
[the remark this answers is not in the document]
Comment (vishalgupta-eGov) [reply backend/expense-calculator/src/main/java/org/egov/digit/expense/calculator/util/MdmsUtils.java:36]: This constant for MDMS PayerList data.

File: backend/expense-calculator/src/main/java/org/egov/digit/expense/calculator/util/ExpenseCalculatorServiceConstants.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (subhashini-egov) [reply backend/expense-calculator/src/main/java/org/egov/digit/expense/calculator/util/ExpenseCalculatorServiceConstants.java]: @vishalgupta-eGov we don't need to send status. Please remove that.

**Pull request merged**